roman-sam/server.js

```diff
@@ -0,0 +1,193 @@
+client.connect();
+
+
+// REVIEW: Install the middleware plugins so that our app can parse the request body
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(express.static('./public'));
+
+
+// REVIEWED: Routes for requesting HTML resources
+app.get('/new-article', (request, response) => {
+  // COMMENTED: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js, if any, is interacting with this particular piece of `server.js`? What part of CRUD, if any, is being enacted/managed by this particular piece of code?
+  // #5 - response
+  // We see no method used in article.js for this function - instead, it happens when the user types /new in the URL bar to access the form for a new article.
+  // CRUD is read
+  response.sendFile('new.html', {root: './public'});
+});
+
+
+// REVIEWED: Routes for making API calls to use CRUD Operations on our database
+app.get('/articles', (request, response) => {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  //#3 query
+  // fetchAll 
+  // CRUD - get
+
+  client.query('SELECT * FROM articles;')
+    .then(function(result) {
+      response.send(result.rows);
+    })
+    .catch(function(err) {
+      console.error(err)
+    })
+});
+
+app.post('/articles', (request, response) => {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  // #4 -
+  // 
+  // CRUD = PUT
+  let SQL = `
+    INSERT INTO articles(title, author, "authorUrl", category, "publishedOn", body)
+    VALUES ($1, $2, $3, $4, $5, $6);
+  `;
+
+  let values = [
+    request.body.title,
+    request.body.author,
+    request.body.authorUrl,
+    request.body.category,
+    request.body.publishedOn,
+    request.body.body
+  ]
+
+  client.query( SQL, values )
+    .then(function(data) {
+      // console.log('LINE 75 s/ays this -- insert complete', data)
+      response.send('LINE 75 says this -- insert complete')
+    })
+    .catch(function(err) {
+      console.error(err);
+    });
+});
+
+app.put('/articles/:id', (request, response) => {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  // PUT YOUR RESPONSE HERE
+
+  let SQL = 'UPDATE articles SET author = $1, "authorUrl" = $2, body = $3, category = $4, "publishedOn" = $5, title = $6 WHERE article_id = $7' ;
+  let values = [request.body.author, 
```

Comment:
It looks like you didn't have eslint set up correctly; there are lots of errors around these lines. Make sure you have eslint set up on every project.